orc8r/cloud/deploy/terraform/orc8r-aws/variables.tf

```diff
@@ -303,6 +303,12 @@ variable "orc8r_db_event_subscription" {
   default     = "orc8r-rds-events"
 }
 
+variable "orc8r_db_apply_immediately"{
+  description = "Apply DB changes immediately or after during maintenance window"
```

Comment:
This wording is a little confusing to me

Reply:
perhaps reword it as "Flag to immediately upgrade RDS without waiting for a maintenance window" ?

Reply:
That makes sense / is clear to me 👍

Reply:
Updated.